cli/pkg/pgx/pgx.go

```diff
@@ -1,11 +1,13 @@
-package cmd
+package pgx
```

Comment:
Can't say I love this package name since it clashes with jackc/pgx. Maybe we can name it pgconn?